csm/core/repositories/alerts.py

```diff
@@ -123,7 +123,7 @@ async def count_by_range(self, time_range: DateTimeRange):
     #         "severity": 1,
     #         "state": "fault_resolved",
     #         "extended_info": {
-    #             "resource_type": "fru",
+    #             "resource_type": "hw",
```

Comment:
You have made changes to a commented code. Instead remove the commented code

Reply:
The commented code written for **to dump the data while starting the server** even we can delete that commented code?

Reply:
Remove the commented code